File: assignment 10 - C# practice problems/11.cs
//Write a c# program to convert the letters of a given 
//stringInputing in alphabetical order.
using System;
namespace MyNameSpace
{
    class Program11
    {
        public static void StringInAscendingOrderDriverFunction()
        {

            Console.WriteLine("Enter a string: ");
            string inputSrring = Console.ReadLine();

            var inpurArray = inputSrring.ToCharArray();
            Array.Sort(inpurArray);

            Console.WriteLine(String.Join("", inpurArray));
        }
    }
}

File: assignment 10 - C# practice problems/js/11.js
//Write a JavaScript program to convert the letters of a given 
//stringInputing in alphabetical order.
function stringInAscendingOrderDriverFunction() {
    let stringInput = prompt("Enter a stringInputing");

    if (stringInput != null) {
        stringInput = stringInput.split("");

        stringInput.sort();

        alert(stringInput.join(""));
    }
}

stringInAscendingOrderDriverFunction()